cibyl/sources/zuuld/errors.py

```diff
@@ -0,0 +1,31 @@
+"""
+#    Copyright 2022 Red Hat
+#
+#    Licensed under the Apache License, Version 2.0 (the "License"); you may
+#    not use this file except in compliance with the License. You may obtain
+#    a copy of the License at
+#
+#         http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#    WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#    License for the specific language governing permissions and limitations
+#    under the License.
+"""
+
+
+class ZuulDError(Exception):
```

Comment:
This should inherit from `SourceException` to easily catch errors raised by this source

Reply:
ZuulD is not really a source. A expect that the Zuul source will be catch this kind of error and return them as a SourceException. On any case I will make the change as it won't break anything.

Reply:
Done.